frontend/src/editor/player.tsx

```diff
@@ -173,6 +173,8 @@ export function PlayerBar({ documentId, editor }: { documentId: string; editor:
             <WaveForm id="waveform" cursorColor="red" barWidth={2} normalize responsive={true} />
           </WaveSurfer>
         </div>
+
+        <PlaybackSpeedDropdown onChange={(v) => waveSurferRef.current?.setPlaybackRate(v)} />
```

Comment:
Maybe we should also switch the wavesurfer `backend` to `MediaElement` to the media gets pitch corrected https://github.com/wavesurfer-js/wavesurfer.js/issues/149#issuecomment-310880071

Reply:
sounds like a good idea 👍

Reply:
Yes!

Reply:
> Does this also fix the jumping cursor position, when the speed is changed?

Seems not to be the case :(

Reply:
@anuejn Just noticed the backend was not changed to `MediaElement` is there a reason for this?